### ISSUE TITLE
AHAP Lineage missing some character faces

### ISSUE THREAD

**Issue by connorhsm** [jasonrohrer/OneLife#1100]
http://lineage.onehouronelife.com/ahapLineageServer/server.php?action=front_page

![image](https://github.com/jasonrohrer/OneLife/assets/20902771/1e9fa3e7-94bc-48ec-8faf-506865402d96)

**Comment by jasonrohrer** [comment]
Hmm... interesting.... these must be new characters that have been added.

I'll see what's going on here.

**Comment by jasonrohrer** [comment]
Yeah, it looks like the new faces are not being committed by the content leader, for some reason.  Maybe one of the scripts is missing that.

**Comment by jasonrohrer** [comment]
Looks like this might just be an authorship issue:

https://github.com/jasonrohrer/AnotherPlanetData/issues/12

**Comment by MarkLindsay** [comment]
> Looks like this might just be an authorship issue:
> 
> [jasonrohrer/AnotherPlanetData#12](https://github.com/jasonrohrer/AnotherPlanetData/issues/12)

Fixed on AHAP side.
